### ISSUE TITLE
Big bug: cronopete move home folder to another name

### ISSUE THREAD

**Issue by maieul** [rastersoft/cronopete#52]
Hi,

just some minutes ago, it happens a strange thing.
Cronopete start it backup. But instead of rsync <code>/home/username</code> to the backup device, it move it to <code>/home/2018_10_01_18:12:34:15384135</code>.

Consequently : not any program was able to find its file. I created in emergency an user, restart, and opening session of this user, I see tha
![capture_du_2018-10-01_18-32-23](https://user-images.githubusercontent.com/636640/46302582-ae58cc00-c5a9-11e8-88f6-5290114f2439.png)
t the files was kept (cf screenshot).

**Comment by rastersoft** [comment]
What????????? That makes no sense...

Which backend are you using?

**Comment by maieul** [comment]
That what I think first, but I can ensure you that happens, when  cronopete start it process... I was a big afraid. 
I am on Ubuntu 18.04 LTS.

**Comment by rastersoft** [comment]
No, no... which backend, in cronopete. Are you using the default rsync backend for an external hard disk, or are you using the backend for storing the backups in a folder?

**Comment by rastersoft** [comment]
Does it happens always, or only once?

**Comment by maieul** [comment]
only once, this afternoon.

I have used cronopete since one year, and it didn't happens in the past

**Comment by rastersoft** [comment]
And which backend, in cronopete. Are you using the default rsync backend for an external hard disk, or are you using the backend for storing the backups in a folder?

**Comment by maieul** [comment]
to store in an external disk.

**Comment by rastersoft** [comment]
I will check all the places where I rename folders... but it is very odd...

**Comment by maieul** [comment]
yeah...

**Comment by maieul** [comment]
I use cronopete 4.4.0

**Comment by rastersoft** [comment]
Please, update to version 4.6.0.

**Comment by maieul** [comment]
done

**Comment by rastersoft** [comment]
Ok. It fixes a problem when there are blank spaces. Just in case. Anyway, I'm checking the code.

**Comment by maieul** [comment]
thanks.

No space in my disk name.

**Comment by rastersoft** [comment]
Yes, but the bug triggered by that could also be triggered by other "odd elements", so just to be sure...

**Comment by maieul** [comment]
same problem today. Maybe the problem happens when the external hard disk is disconnected for any reason when processing to the backup.
